components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -49,6 +51,19 @@ pub fn ascii(arg: &Option<Bytes>) -> Result<Option<i64>> {
     }))
 }
 
+#[rpn_fn]
+#[inline]
+pub fn rtrim(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().map(|bytes| {
+        let pos = bytes.iter().rev().position(|&x| x != SPACE);
```

Comment:
How about using `rposition`?

Reply:
cool, that make it more clearer, thx  ;-)